cmd/root.go

```diff
@@ -125,31 +131,41 @@ var rootCmd = &cobra.Command{
 		}
 
 		if !checkFileExists(filePath) {
-			log.Fatal("File ", filePath, " does not exist")
+			return fmt.Errorf("file %s does not exist", filePath)
```

Comment:
`errors.Wrap` here and on all the ones below?

Reply:
This one does not have an err to wrap, but the rest of them are updated